tests/unit/test_tag_chartbeat.py

```diff
@@ -8,39 +8,46 @@
 from django.template import Context
 from django.test import TestCase
 from django.test.utils import override_settings
-
-from analytical.templatetags.chartbeat import ChartbeatTopNode, \
-        ChartbeatBottomNode
 from utils import TagTestCase
+
+from analytical.templatetags.chartbeat import ChartbeatBottomNode, ChartbeatTopNode
 from analytical.utils import AnalyticalException
 
 
-@override_settings(CHARTBEAT_USER_ID='12345')
+@override_settings(CHARTBEAT_USER_ID="12345")
 class ChartbeatTagTestCaseNoSites(TestCase):
     def test_rendering_setup_no_site(self):
         r = ChartbeatBottomNode().render(Context())
         self.assertTrue('var _sf_async_config={"uid": "12345"};' in r, r)
 
 
-@override_settings(INSTALLED_APPS=(
-    'analytical',
-    'django.contrib.sites',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-))
-@override_settings(CHARTBEAT_USER_ID='12345')
+@override_settings(
+    INSTALLED_APPS=(
+        "analytical",
+        "django.contrib.sites",
+        "django.contrib.auth",
+        "django.contrib.contenttypes",
+    )
```

Comment:
This should be a list in current Django versions.

```suggestion
    INSTALLED_APPS=[
        "analytical",
        "django.contrib.sites",
        "django.contrib.auth",
        "django.contrib.contenttypes",
    ]
```